webclient/src/websocket/commands/session/resetPasswordRequest.ts

```diff
@@ -0,0 +1,41 @@
+import { ForgotPasswordParams } from "store";
+import { StatusEnum, WebSocketConnectOptions } from "types";
+
+import webClient from "../../WebClient";
+import { SessionPersistence } from "../../persistence";
+
+import { disconnect, updateStatus } from "./";
+
+export function resetPasswordRequest(options: WebSocketConnectOptions): void {
+  const { userName } = options as unknown as ForgotPasswordParams;
+
+  const forgotPasswordConfig = {
+    ...webClient.clientConfig,
+    userName,
+  };
+
+  const CmdForgotPasswordRequest = webClient.protobuf.controller.Command_ForgotPasswordRequest.create(forgotPasswordConfig);
+
+  const sc = webClient.protobuf.controller.SessionCommand.create({
+    '.Command_ForgotPasswordRequest.ext': CmdForgotPasswordRequest
+  });
+
+  webClient.protobuf.sendSessionCommand(sc, raw => {
+    if (raw.responseCode === webClient.protobuf.controller.Response.ResponseCode.RespOk) {
```

Comment:
Would it make more sense to do a negative case if (code != okay) and do a quick exit? Would remove indentation and might clear things up to avoid so many nested conditionals